### PR TITLE
feat(ord-connect): add okx-wallet to ord-connect

### PR DESCRIPTION
**Pull request by danadi7**
#### What this PR does / why we need it:
- Adds OKX Wallet to ord-connect

Screenshot 2024-03-12 at 14 22 31

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors on web
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: packages/ord-connect/src/components/SelectWalletModal/index.tsx [hunk 1]
```diff
@@ -312,6 +315,46 @@ export function SelectWalletModal({
...
+  const onConnectOKXWallet = useCallback(async () => {
+    try {
+      setErrorMessage("");
+      const okx = await getOKXAddresses(network);
+      if (!okx || okx.length < 1) {
+        disconnectWallet();
+        throw new Error("OKX via Ordit returned no addresses.");
+      }
+
+      const okxwallet = okx[0];
```
Comment (Nanosync) [inline packages/ord-connect/src/components/SelectWalletModal/index.tsx:327]:
```suggestion
      const okxWallet = okx[0];
```

**Review by kevzzsk — COMMENTED** [review]
lgtm

File: packages/ord-connect/src/lib/signPsbt.ts [hunk 2]
```diff
@@ -95,6 +96,23 @@ export default async function signPsbt({
...
+
+  if (wallet === Wallet.OKX) {
+    const signedOKXPsbt = await signOKXPsbt(psbt, {
+      finalize,
+      extractTx,
+      network,
+      inputsToSign: [
+        {
+          address,
+          signingIndexes: options?.signingIndexes ?? getAllInputIndices(), // If signingIndexes is not provided, just sign everything
```
[the remark this answers is not in the document]
Comment (danadi7) [reply packages/ord-connect/src/lib/signPsbt.ts:108]: Removed

File: packages/ord-connect/src/components/SelectWalletModal/index.tsx (hunk 1 above)
Comment (danadi7) [reply packages/ord-connect/src/components/SelectWalletModal/index.tsx]: Fixed

**Review by Nanosync — APPROVED** [review]

**Pull request merged**